recipes/mvapich2/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -ex
+
+unset F77 F90
+
+export FFLAGS="-fallow-argument-mismatch ${FFLAGS}"
+
+export CC=$(basename "$CC")
+export CXX=$(basename "$CXX")
+export FC=$(basename "$FC")
+
+build_with_rdma=""
+if [[ "$target_platform" == linux-* ]]; then
```

Comment:
From the `skips`, this condition is always true, right?

Reply:
I plan to do it for other target platforms, eventually, so I would rather leave that in place

Reply:
Makes sense, got it.